### ISSUE TITLE
Theming App Problem

### ISSUE THREAD

**Issue by ExaconAT** [nextcloud/server#9479]
### Steps to reproduce
1. Admin -> Theming
2. Change BG Colour and Header Picture x 3

### Expected behaviour

- 1 Header Picture
- 1 Colour

### Actual behaviour
Randomly generated Colors appear (the last selected ones + nc default color) and sometimes the Header Picture ist the Nextcloud

### Server configuration

**Operating system**: 16.04.4

**Web server:** Apache/2.4.25 (Debian)

**Database:** mariadb  Ver 15.1 Distrib 10.1.26-MariaDB

**PHP version:** PHP 7.0.27-0+deb9u1

**Nextcloud version:** 13.0.2

**Updated from an older Nextcloud/ownCloud or fresh install:** Updated because Webinstaller was outdated (13.0.1? -> 13.0.2)

**Where did you install Nextcloud from:** Nextcloud.com Webinstaller

**Signing status:**
<details>
<summary>Signing status</summary>

</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - activity: 2.6.1
  - bruteforcesettings: 1.0.3
  - comments: 1.3.0
  - dav: 1.4.6
  - federatedfilesharing: 1.3.1
  - federation: 1.3.0
  - files: 1.8.0
  - files_pdfviewer: 1.2.1
  - files_sharing: 1.5.0
  - files_texteditor: 2.5.1
  - files_trashbin: 1.3.0
  - files_versions: 1.6.0
  - files_videoplayer: 1.2.0
  - firstrunwizard: 2.2.1
  - gallery: 18.0.0
  - impersonate: 1.0.4
  - logreader: 2.0.0
  - lookup_server_connector: 1.1.0
  - nextcloud_announcements: 1.2.0
  - notifications: 2.1.2
  - oauth2: 1.1.0
  - password_policy: 1.3.0
  - provisioning_api: 1.3.0
  - serverinfo: 1.3.0
  - sharebymail: 1.3.0
  - survey_client: 1.1.0
  - systemtags: 1.3.0
  - theming: 1.4.1
  - twofactor_totp: 1.4.1
  - unsplash: 1.1.1
  - updatenotification: 1.3.0
  - workflowengine: 1.3.0
Disabled:
  - admin_audit
  - encryption
  - files_external
  - user_external
  - user_ldap
  - rainloop: 5.0.6
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
t system
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "192.168.1.146",
            "nextcloud.fruchtig.cc"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/nextcloud.fruchtig.cc",
        "dbtype": "mysql",
        "version": "13.0.2.1",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "mail_smtpmode": "smtp",
        "mail_smtpauthtype": "LOGIN",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379
        },
        "theme": "",
        "loglevel": 2,
        "maintenance": false,
        "mail_smtpsecure": "tls",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauth": 1,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "587",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***"
    }
}

```
</details>

### Client configuration
**Browser:** Firefox newest, Edge newest

**Operating system:** Win 10

#### Web server error log
<details>
<summary>Web server error log</summary>

```
EMPTY
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
Nothing special
```
</details>
</details>

**Comment by MorrisJobke** [comment]
cc @nextcloud/theming

**Comment by juliushaertl** [comment]
Can you rephrase/extend the steps to reproduce a bit? Are you seeing the Saved indicator in the theming app settings? Are you making the changes quickly after each other?

**Comment by MorrisJobke** [comment]
As there is no feedback since a while I will close this ticket. If this is still happening please feel free to reopen.
